test/Microsoft.AspNetCore.OData.Versioning.ApiExplorer.Tests/AspNet.OData/Contact.cs

```diff
@@ -1,18 +1,25 @@
-﻿namespace Microsoft.AspNet.OData
+﻿using System.Runtime.Serialization;
+
+namespace Microsoft.AspNet.OData
 {
     using System.Collections.Generic;
-
+    [DataContract]
     public class Contact
     {
+        [DataMember]
         public int ContactId { get; set; }
-
+        [DataMember(Name = "first_name")]
         public string FirstName { get; set; }
+        [DataMember]
 
         public string LastName { get; set; }
+        [DataMember]
 
         public string Email { get; set; }
+        [DataMember]
 
         public string Phone { get; set; }
+        [DataMember]
 
         public List<Address> Addresses { get; set; }
```

Comment:
The `[DataMember]` attribute is misaligned. It should have the line above it, not below it. This should be a quick fix. Thanks.